active_directory/datadog_checks/active_directory/metrics.py

```diff
@@ -0,0 +1,169 @@
+    [
+        'NTDS',
+        None,
+        'DRA Inbound Full Sync Objects Remaining',
+        'active_directory.dra.inbound.objects.remaining',
+        'gauge',
+    ],
+    ['NTDS', None, 'DRA Inbound Objects/sec', 'active_directory.dra.inbound.objects.persec', 'gauge'],
+    ['NTDS', None, 'DRA Inbound Objects Applied/sec', 'active_directory.dra.inbound.objects.applied_persec', 'gauge'],
+    ['NTDS', None, 'DRA Inbound Objects Filtered/sec', 'active_directory.dra.inbound.objects.filtered_persec', 'gauge'],
+    [
+        'NTDS',
+        None,
+        'DRA Inbound Object Updates Remaining in Packet',
+        'active_directory.dra.inbound.objects.remaining_in_packet',
+        'gauge',
+    ],
+    [
+        'NTDS',
+        None,
+        'DRA Inbound Properties Applied/sec',
+        'active_directory.dra.inbound.properties.applied_persec',
+        'gauge',
+    ],
+    [
+        'NTDS',
+        None,
+        'DRA Inbound Properties Filtered/sec',
+        'active_directory.dra.inbound.properties.filtered_persec',
+        'gauge',
+    ],
+    ['NTDS', None, 'DRA Inbound Properties Total/sec', 'active_directory.dra.inbound.properties.total_persec', 'gauge'],
+    ['NTDS', None, 'DRA Inbound Values (DNs only)/sec', 'active_directory.dra.inbound.values.dns_persec', 'gauge'],
+    ['NTDS', None, 'DRA Inbound Values Total/sec', 'active_directory.dra.inbound.values.total_persec', 'gauge'],
+    [
+        'NTDS',
+        None,
+        'DRA Outbound Bytes Compressed (Between Sites, After Compression)/sec',
+        'active_directory.dra.outbound.bytes.after_compression',
+        'gauge',
+    ],
+    [
+        'NTDS',
+        None,
+        'DRA Outbound Bytes Compressed (Between Sites, Before Compression)/sec',
+        'active_directory.dra.outbound.bytes.before_compression',
+        'gauge',
+    ],
+    [
+        'NTDS',
+        None,
+        'DRA Outbound Bytes Not Compressed (Within Site)/sec',
+        'active_directory.dra.outbound.bytes.not_compressed',
+        'gauge',
+    ],
+    ['NTDS', None, 'DRA Outbound Bytes Total/sec', 'active_directory.dra.outbound.bytes.total', 'gauge'],
+    [
+        'NTDS',
+        None,
+        'DRA Outbound Objects Filtered/sec',
+        'active_directory.dra.outbound.objects.filtered_persec',
+        'gauge',
+    ],
+    ['NTDS', None, 'DRA Outbound Objects/sec', 'active_directory.dra.outbound.objects.persec', 'gauge'],
+    ['NTDS', None, 'DRA Outbound Properties/sec', 'active_directory.dra.outbound.properties.persec', 'gauge'],
+    ['NTDS', None, 'DRA Outbound Values (DNs only)/sec', 'active_directory.dra.outbound.values.dns_persec', 'gauge'],
+    ['NTDS', None, 'DRA Outbound Values Total/sec', 'active_directory.dra.outbound.values.total_persec', 'gauge'],
+    # ['NTDS', None, 'DRA Remaining Replication Updates', 'active_directory.dra.replication.remaining_updates', 'gauge'],  # noqa: E501
```

Comment:
I know these were all commented out before, but is there documentation on why? If they don't exist then we can remove these comments, if we couldn't collect them before maybe we can collect them now?